### PR TITLE
Meetup creation / Pins icon selection

### PR DESCRIPTION
**Pull request by TruddyTheDuddi**
**What**
Small design fix for the meetup creation

- Designed a new icon for the icons selector
- Changed text and hint text for icons
- Now user clicks on the Layout instead of a button

### CODE REVIEW

File: app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpCreationViewModel.kt [hunk 1]
```diff
@@ -111,6 +111,7 @@ class MeetUpCreationViewModel @Inject constructor(
         _tags.value = emptySet()
         _participantsId.value = listOf(profileService.getLoggedInUserID()!!)
         _criterionAge.value = Pair(13, 66)
```
Comment (Gerussi) [inline app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpCreationViewModel.kt:113]: We could move those into the integers.xml file (I know not part of your work but I notice it now

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -207,5 +207,7 @@
     <string name="scanned">Scanné</string>
     <string name="no_camera_permission_message">Annulé à cause des permissions de la caméra</string>
     <string name="cancelled_scan">Scan annulé</string>
+    <string name="meetup_creation_icon_title">Appyuer pour choisir le marker</string>
```
Comment (Gerussi) [inline app/src/main/res/values-fr/strings.xml:210]: Small typo, also could translate marker into repère

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -207,5 +207,7 @@
...
+    <string name="meetup_creation_icon_hint">Changez l\'icone de la rencontre directement sur la map</string>
```
Comment (Gerussi) [inline app/src/main/res/values-fr/strings.xml:211]: map peut être traduit par carte

**Review by atappy — APPROVED** [review]
mainly xml changes, good job anyway !

**Pull request merged**